prime-router/src/main/kotlin/history/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -38,19 +42,15 @@ interface SubmissionAccess {
         klass: Class<T>
     ): List<T>
 
-    fun <T, P, U> fetchAction(
+    fun <T> fetchAction(
```

Comment:
Add kdoc describing what this is doing. Understandable this is not code you originally wrote, but please document it in this PRc

Reply:
done!